pkgs/tools/networking/burpsuite/professional.nix

```diff
@@ -42,8 +41,8 @@ stdenv.mkDerivation rec {
     homepage = "https://portswigger.net/burp/";
     downloadPage = "https://portswigger.net/burp/freedownload";
     license = licenses.unfree;
-    platforms = jdk11.meta.platforms;
+    platforms = openjdk.meta.platforms;
     hydraPlatforms = [];
-    maintainers = with maintainers; [ bennofs ];
+    maintainers = with maintainers; [ madonius ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ madonius bennofs ];
```

Reply:
@ckiee as @bennofs does not maintain the professional version, I am unsure whether this would be correct.

Reply:
Oh oops, `community.nix` then.